backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,393 @@
+
+—------------------------------------------------------------------------------------------------------------------------
+TEST #1 - 100-105
+Get active users
+
+***
+Pre-req: -
+***
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = hash
+
+    }
+
+    SQL-query: SELECT active_users FROM groupdugga WHERE hash=hash
+    This should show all active users
+
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #2 - 106-110
+Create active users
+
+***
+Pre-req: $active == NULL
+***
+
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = “tj7dh2nb”
+        $AUtoken = 999
+    }
+
+    SQL-query: INSERT INTO groupdugga(hash,active_users) VALUES(:hash,:AUtoken)
+***
+* Get newly added active users
+***
+	SQL-query: SELECT MAX(active_users) FROM groupdugga; // Save to variable $active_users this will identify the last active user that was created (it has the highest AUtoken)
+
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+***
+* Remove inserted row from DB, using hash
+***
+
+	SQL-query DELETE FROM groupdugga WHERE hash = $hash;
+
+—------------------------------------------------------------------------------------------------------------------
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #3 - 111-117
+update active users
+
+***
+Pre-req: $active != NULL
+***
+
+***
+Pre-req variables
+***
+$pre-values =
+    {
+        $hash = “hjk4ert6”
+        $AUtoken = 52
+    }
+
+    SQL-query: INSERT INTO groupdugga(hash,active_users) VALUES($pre-values[0], $pre-values[1])
+
+***
+* Get newly created active users, 
+***
+	SQL-query: SELECT * FROM groupdugga WHERE hash=$pre-values[0]; //
+
+
+***
+* Update information on a active users
+* send parameters input to showDuggaservice.php
+***
+send
+    {
+        $opt = “UPDATEAU”
+        $hash = “hjk4ert6”
+        $newToken = 9999
+    }
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+***
+* Remove inserted row from DB, using hash
+***
+
+	SQL-query: DELETE FROM groupdugga WHERE hash = $hash;
+
+—------------------------------------------------------------------------------------------------------------------
+—------------------------------------------------------------------------------------------------------------------
+***
+Test#4 - 261 - 309
```

Comment:
weren't we suppose to only cover the services? 

ill go trough the rest ether way, but this might be unneeded for the issue...

Reply:
![image](https://user-images.githubusercontent.com/102668230/234826695-58251051-a414-4565-939d-359549f96533.png)

Reply:
ah.. sorry my bad, the structure of this file is a little all over the place.